text_to_speech_MOD.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  name: 'Text To Speech',
+  section: 'Messaging',
+  
+  subtitle: function(data) {
+    const info = ['Item 1', 'Item 2', 'Item 3'];
```

Comment:
What is this line for? You don’t use `info` anywhere

Reply:
removed